kserve-rest-proxy.yaml

```diff
@@ -19,6 +19,14 @@ pipeline:
       packages: ./proxy
       output: rest-proxy
 
+subpackages:
+  - name: kserve-rest-proxy-compat
+    pipeline:
+      - name: Link proxy binary to expected path in upstream image
+        runs: |
+          mkdir -p "${{targets.contextdir}}/go/bin"
+          ln -s "${{targets.destdir}}/usr/bin/rest-proxy" "${{targets.contextdir}}/go/bin/server"
```

Comment:
This would be `ln -s "/usr/bin/rest-proxy" "${{targets.contextdir}}/go/bin/server"`